src/Plot.tsx

```diff
@@ -137,12 +137,13 @@ export default function Plot({
   }, [state, bottom, top, height]);
 
   const labels = state.series.map(({ label }) => label);
+  const ids = state.series.map(({ id }) => id);
```

Comment:
it's better to put this inside the useMemo, otherwise the `ids` array will always be a new array if it's created here